code/game/verbs/rolldice.dm

```diff
@@ -0,0 +1,9 @@
+/client/verb/rolldice()

```

Comment:
This proc should be moved into `code/game/verbs/ooc.dm`. It does not need its own file.

Reply:
Also I would just update admin verb and make it check if user has admin perms:
 - If user has admin perms we prompt them if they want to broadcast it.
 - If user has no admin perms  we just roll it and tell user only.

Reply:
Right on. I moved it to ooc.dm, for one at least.